style.css

```diff
@@ -0,0 +1,66 @@
+.headline {
+  background-image: url("./images/header-illsutration-mobile.svg");
+  background-repeat: no-repeat;
+  background-size: 100%;
+  position: absolute;
+}
+
+.personal-presentation h1 {
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  text-align: center;
+  line-height: 52px;
+  font-style: normal;
+  margin-top: 58px;
+  margin-left: 27px;
+  margin-right: 27px;
+  margin-bottom: 0%;
+}
+
+.personal-presentation p {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 19px;
+  text-align: center;
+  line-height: 28px;
+  font-style: normal;
+  margin-left: 27px;
+  margin-right: 27px;
+  margin-top: 15px;
+}
+
+.social-media ul {
+  list-style: none;
+}
+
+.color-blue {
+  color: #172b4d;
+}
+
+#title-green {
+  color: #36b37e;
+}
+
+nav {
+  display: flex;
+  justify-content: end;
+}
+
+nav img {
+  margin-right: 38px;
+  margin-top: 40px;
+}
+
+.social-media-links {
+  display: flex;
```

Comment:
- [ ] I would kindly suggest you work on the suggestion given by the previous reviewer concerning repetition of `display: flex;` multiple times in your CSS as this is against the DRY method of coding practices.

Kindly check the example he gave and try to come up with something fantastic